repo/packages/M/mariadb/3/config.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Please follow the same semantics as the previous mariaDB package: https://github.com/mesosphere/universe/tree/version-3.x/repo/packages/M/mariadb/2